DTM/BE/BE_DTM_tindex.py

```diff
@@ -0,0 +1,160 @@
+############################################################################
+#
+# NAME:         BE_DTM_tindex.py
+#
+# AUTHOR(S):    Anika Weinmann
+#               mundialis GmbH & Co. KG, Bonn
+#               https://www.mundialis.de
+#
+# PURPOSE:      Create tile index of Berlin DGM/DTM xyz files
+#
+# Data source:  https://www.opengeodata.nrw.de/produkte/geobasis/dop/dop/
+#
+# COPYRIGHT:    (C) 2023 by Markus Neteler, Anika Weinmann, mundialis
+#
+# REQUIREMENTS: lynx, gdal, gzip, sed
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+############################################################################
+# TODO
+# Usage:
```

Comment:
```suggestion
# Usage:
# Set Parameter `GDALTINDEX`
# `GDALTINDEX=True`: using `gdaltindex` (more correct limits of the tiles)
# `GDALTINDEX=False`: creating the tiles out of the file names (only small improvement in runtime, because you still have to check with `gdalinfo` whether the tiles are valid files at all )
# Then call script like this:
```